repos/spack_repo/builtin/packages/nseg/package.py

```diff
@@ -19,46 +19,53 @@ class Nseg(MakefilePackage):
         expand=False,
     )
 
+    source_files = ["makefile", "genwin.c", "genwin.h", "lnfac.h", "nmerge.c", "runnseg"]
```

Comment:
It would be great if someone could convince the package authors that this is an absurd way to provide software...

In the meantime, can you turn this into a tuple of `(name, sha256)` for each file? Then the repeated resource directives can be replaced with:

```
for name, sha256 in source_files:
    resource(
        name=name,
        url=f"ftp://ftp.ncbi.nih.gov/pub/seg/nseg/{name}",
        sha256=sha256,
        expand=False,
        placement=name,
    )
```